clio-client/src/main/scala/org/broadinstitute/clio/client/metadata/CramDeliverer.scala

```diff
@@ -37,7 +37,7 @@ class CramDeliverer extends MetadataMover[CramMetadata] {
       writeMd5Op.toIterable
     ).flatten
 
-    if (src.regulatoryDesignation.exists(_.equals(RegulatoryDesignation.ResearchOnly))) {
+    if (deliverSampleMetrics) {
```

Comment:
We should talk about where we want to the check for regulatory designation to happen. Doing it in the delivery WDL will be kinda unwieldy. We don't have the Regulatory Designation available on a per-sample basis when we call `MoveSamples` in the WDL to make the decision of whether or not to actually move sample level metrics.

Reply:
Why not just keep it here?

Reply:
The original JIRA says
> All sample-level metrics files should be delivered for most samples but not for clinical samples or for fee-for-service projects
Since we don't have a way to test for fee-for-service (and since we don't know yet how users will request this delivery feature) I thought it best to only use the command line argument to toggle this setting.

Reply:
We know fee-for-service when delivery is started, so that flag can be set easily in the WDL. We don't know the Regulatory Designation of each sample in the WDL though, and its much more effort to get that into the WDL than to just check for it when we're already in Clio.

Reply:
OK, I can add it back in. The original issue says `not for clinical samples` but the code you added tests for `ResearchOnly`. Is there a reason why you didn't use `!Clinical`?